.github/workflows/releases.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package to PyPI
+
+on:
+  push:
+    branches: [stable]
+  release:
+    types: [published]
+
+jobs:
+  lint-and-unit-tests:
+    name: Lint & Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox
+    - name: Run lint checkers
+      run: |
+        tox -e lint
+    - name: Run unit tests
+      run: |
+        tox -e unit
+
+  release-to-pypi:
+    name: Release juju-verify to PyPI
+    runs-on: ubuntu-latest
+    needs: lint-and-unit-tests
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox
+    - name: Build package
+      run: tox -e build
```

Comment:
Can we us ${{ github.ref }} as version? Because such a version must be hard-coded, and if I want to create a new release, I also need to change the setup.cfg file.
e.g.
```python
# setup.py
setup(version=os.environ.get("JUJU-VERIFY-VERSION", "dev"))
```

Reply:
We went through it internally.